homeassistant/components/wallbox/sensor.py

```diff
@@ -169,17 +175,6 @@ def __init__(
         self._attr_name = f"{entry.title} {description.name}"
 
     @property
-    def native_value(self) -> StateType:
+    def native_value(self) -> int | float | str:
         """Return the state of the sensor."""
-        if (sensor_round := self.entity_description.precision) is not None:
```

Comment:
I prefer the old way -> doing the round in one place, we could replace it with a lambda of course.